src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -71,6 +70,7 @@ private async Task UpdateMarkdownAsync()
         {
             try
             {
+                descriptionMarkdownPreview.Content = descriptionMarkdownPreview.Content ?? _markdownPreview.VisualElement;
```

Comment:
Question: Is this related to this particular change or does this address something else?

Reply:
Sort of related. The loading dialog was shown large on initial load, it seemed to be related to when the content for descriptionMarkdownPreview was being set. The datacontext for the view isn't set in the construction of the view. So I moved where I sent the content to the update function since that's the only time it needs to be set anyway, and I made sure to only set it once.